# Person msg giving information about a person.
Header header
Person[] persons
